Build currently fails in Fedora-36 and higher

The build currently fails in Fedora-36 and Fedora-Rawhide (compiled with gcc-12.0.1 20220125)

The errors are shown below:
```console
+ mamake -C cmd/ksh93 'KSH_RELFLAGS=-D_AST_git_commit=\"abc7f04d\"' 'KSH_SHOPTFLAGS=-DSHOPT_ACCT=0 -DSHOPT_ACCTFILE=0 -DSHOPT_AUDIT=1 -DSHOPT_AUDITFILE=\"/etc/ksh_audit\" -DSHOPT_BGX=1 -DSHOPT_BRACEPAT=1 -DSHOPT_DYNAMIC=1 -DSHOPT_EDPREDICT=0 -DSHOPT_ESH=1 -DSHOPT_FILESCAN=1 -DSHOPT_FIXEDARRAY=1 -DSHOPT_HISTEXPAND=1 -DSHOPT_MKSERVICE=0 -DSHOPT_MULTIBYTE=1 -DSHOPT_NAMESPACE=1 -DSHOPT_NOECHOE=0 -DSHOPT_OPTIMIZE=1 -DSHOPT_RAWONLY=1 -DSHOPT_STATS=1 -DSHOPT_SUID_EXEC=1 -DSHOPT_VSH=1' install
+ cc '-D_AST_git_commit="abc7f04d"' -DSHOPT_ACCT=0 -DSHOPT_ACCTFILE=0 -DSHOPT_AUDIT=1 '-DSHOPT_AUDITFILE="/etc/ksh_audit"' -DSHOPT_BGX=1 -DSHOPT_BRACEPAT=1 -DSHOPT_DYNAMIC=1 -DSHOPT_EDPREDICT=0 -DSHOPT_ESH=1 -DSHOPT_FILESCAN=1 -DSHOPT_FIXEDARRAY=1 -DSHOPT_HISTEXPAND=1 -DSHOPT_MKSERVICE=0 -DSHOPT_MULTIBYTE=1 -DSHOPT_NAMESPACE=1 -DSHOPT_NOECHOE=0 -DSHOPT_OPTIMIZE=1 -DSHOPT_RAWONLY=1 -DSHOPT_STATS=1 -DSHOPT_SUID_EXEC=1 -DSHOPT_VSH=1 -Os -L. -L/root/ksh/arch/linux.i386-64/lib -o ksh pmain.o libshell.a /root/ksh/arch/linux.i386-64/lib/libdll.a /root/ksh/arch/linux.i386-64/lib/libast.a /root/ksh/arch/linux.i386-64/lib/libast.a /root/ksh/arch/linux.i386-64/lib/libcmd.a -lutil /root/ksh/arch/linux.i386-64/lib/libast.a -lutil /root/ksh/arch/linux.i386-64/lib/libast.a /root/ksh/arch/linux.i386-64/lib/libast.a /root/ksh/arch/linux.i386-64/lib/libast.a
/usr/bin/ld: libshell.a(init.o): in function `sh_reseed_rand':
init.c:(.text+0x18d0): undefined reference to `remainder'
/usr/bin/ld: libshell.a(streval.o): in function `arith_exec':
streval.c:(.text+0x145b): undefined reference to `pow'
/usr/bin/ld: libshell.a(strdata.o): in function `local_issubnormal':
strdata.c:(.text+0xe2): undefined reference to `__fpclassifyl'
/usr/bin/ld: libshell.a(strdata.o): in function `local_fpclassify':
strdata.c:(.text+0xf5): undefined reference to `__fpclassifyl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x10): undefined reference to `acosl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x28): undefined reference to `acoshl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x40): undefined reference to `asinl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x58): undefined reference to `asinhl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x70): undefined reference to `atanl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x88): undefined reference to `atan2l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0xa0): undefined reference to `atanhl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0xb8): undefined reference to `cbrtl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0xd0): undefined reference to `ceill'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x100): undefined reference to `cosl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x118): undefined reference to `coshl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x130): undefined reference to `erfl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x148): undefined reference to `erfcl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x160): undefined reference to `expl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x178): undefined reference to `exp2l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x190): undefined reference to `exp10l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x1a8): undefined reference to `expm1l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x1c0): undefined reference to `fabsl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x1d8): undefined reference to `fabsl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x1f0): undefined reference to `fdiml'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x238): undefined reference to `floorl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x250): undefined reference to `fmal'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x268): undefined reference to `fmaxl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x280): undefined reference to `fminl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x298): undefined reference to `fmodl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x2c8): undefined reference to `hypotl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x2e0): undefined reference to `ilogbl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x430): undefined reference to `j0l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x448): undefined reference to `j1l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x460): undefined reference to `jnl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x490): undefined reference to `lgammal'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x4a8): undefined reference to `logl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x4c0): undefined reference to `log10l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x4d8): undefined reference to `log1pl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x4f0): undefined reference to `log2l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x508): undefined reference to `logbl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x520): undefined reference to `nearbyintl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x538): undefined reference to `nextafterl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x550): undefined reference to `nexttowardl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x568): undefined reference to `powl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x580): undefined reference to `remainderl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x598): undefined reference to `rintl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x5b0): undefined reference to `roundl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x5c8): undefined reference to `scalbl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x610): undefined reference to `sinl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x628): undefined reference to `sinhl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x640): undefined reference to `sqrtl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x658): undefined reference to `tanl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x670): undefined reference to `tanhl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x688): undefined reference to `tgammal'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x6a0): undefined reference to `truncl'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x6b8): undefined reference to `y0l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x6d0): undefined reference to `y1l'
/usr/bin/ld: libshell.a(strdata.o):(.rodata+0x6e8): undefined reference to `ynl'
collect2: error: ld returned 1 exit status
mamake [cmd/ksh93]: *** exit code 1 making ksh
mamake: *** exit code 1 making cmd/ksh93
mamake: *** exit code 1 making all
package: make failed at Tue Mar  8 17:10:05 CET 2022 in /root/ksh/arch/linux.i386-64
```
Full build log: [log.txt](https://github.com/ksh93/ksh/files/8207811/log3.txt)

Following patch fix the problem:
```diff
diff --git a/src/cmd/builtin/Mamfile b/src/cmd/builtin/Mamfile
index 16974ae5..f6e72610 100644
--- a/src/cmd/builtin/Mamfile
+++ b/src/cmd/builtin/Mamfile
@@ -50,7 +50,7 @@ make install
                        exec - ${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} -I. -I${PACKAGE_ast_INCLUDE} -DERROR_CATALOG=\""builtin"\" -D_PACKAGE_ast -DCMD_STANDALONE=b_pty -c pty.c
                done pty.o generated
                bind -lutil dontcare
-               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} ${LDFLAGS} -lm ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o pty pty.o ${mam_libutil} ${mam_libast} ${mam_libcmd}
+               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o pty pty.o -lm ${mam_libutil} ${mam_libast} ${mam_libcmd}
        done pty generated
        make ${INSTALLROOT}/bin
                exec - if silent test ! -d ${INSTALLROOT}/bin
diff --git a/src/cmd/ksh93/Mamfile b/src/cmd/ksh93/Mamfile
index 3936ff92..e2899523 100644
--- a/src/cmd/ksh93/Mamfile
+++ b/src/cmd/ksh93/Mamfile
@@ -1219,7 +1219,7 @@ make install
                                                        make data/math.tab implicit
                                                        done data/math.tab
                                                done features/math.sh dontcare
-                                               exec - iffe ${IFFEFLAGS} -v -c "${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS}" ref ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -I${PACKAGE_ast_INCLUDE} -I${INSTALLROOT}/include ${mam_libdll} ${mam_libcmd} ${mam_libast} ${mam_libm} ${mam_libnsl} : run features/math.sh ${PACKAGEROOT}/src/cmd/ksh93/data/math.tab
+                                               exec - iffe ${IFFEFLAGS} -v -c "${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS}" ref ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -I${PACKAGE_ast_INCLUDE} -I${INSTALLROOT}/include ${mam_libdll} ${mam_libcmd} ${mam_libast} ${mam_libm} ${mam_libnsl} -lm : run features/math.sh ${PACKAGEROOT}/src/cmd/ksh93/data/math.tab
                                                make ${PACKAGE_ast_INCLUDE}/ast_standards.h implicit
                                                done ${PACKAGE_ast_INCLUDE}/ast_standards.h dontcare
                                                make ${INSTALLROOT}/src/lib/libast/FEATURE/float implicit
@@ -1386,7 +1386,7 @@ make install
                prev +li
                prev ${mam_libsocket}
                prev ${mam_libsecdb}
-               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o ksh pmain.o ${mam_libshell} ${mam_libnsl} ${mam_libast}
+               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o ksh pmain.o -lm ${mam_libshell} ${mam_libnsl} ${mam_libast}
        done ksh generated
        make shcomp
                make shcomp.o
@@ -1408,7 +1408,7 @@ make install
                prev +li
                prev ${mam_libsocket}
                prev ${mam_libsecdb}
-               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o shcomp shcomp.o ${mam_libshell} ${mam_libnsl} ${mam_libast}
+               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o shcomp shcomp.o -lm ${mam_libshell} ${mam_libnsl} ${mam_libast}
        done shcomp generated
        make suid_exec
                make suid_exec.o
@@ -1429,7 +1429,7 @@ make install
                prev +li
                prev ${mam_libsocket}
                prev ${mam_libsecdb}
-               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} -lm ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o suid_exec suid_exec.o ${mam_libast} ${mam_libnsl} ${mam_libast}
+               exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o suid_exec suid_exec.o -lm ${mam_libast} ${mam_libnsl} ${mam_libast}
        done suid_exec generated
        make shell
                prev libshell.a archive
 

```

--- a/src/cmd/builtin/Mamfile
+++ b/src/cmd/builtin/Mamfile
@@ -50,7 +50,7 @@ make install
 			exec - ${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} -I. -I${PACKAGE_ast_INCLUDE} -DERROR_CATALOG=\""builtin"\" -D_PACKAGE_ast -DCMD_STANDALONE=b_pty -c pty.c
 		done pty.o generated
 		bind -lutil dontcare
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} ${LDFLAGS} -lm ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o pty pty.o ${mam_libutil} ${mam_libast} ${mam_libcmd}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o pty pty.o -lm ${mam_libutil} ${mam_libast} ${mam_libcmd}
 	done pty generated
 	make ${INSTALLROOT}/bin
 		exec - if silent test ! -d ${INSTALLROOT}/bin

--- a/src/cmd/builtin/Mamfile
+++ b/src/cmd/builtin/Mamfile
@@ -50,7 +50,7 @@ make install
 			exec - ${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} -I. -I${PACKAGE_ast_INCLUDE} -DERROR_CATALOG=\""builtin"\" -D_PACKAGE_ast -DCMD_STANDALONE=b_pty -c pty.c
 		done pty.o generated
 		bind -lutil dontcare
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o pty pty.o -lm ${mam_libutil} ${mam_libast} ${mam_libcmd}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o pty pty.o ${mam_libutil} ${mam_libast} -lm ${mam_libcmd}
 	done pty generated
 	make ${INSTALLROOT}/bin
 		exec - if silent test ! -d ${INSTALLROOT}/bin

--- a/src/cmd/ksh93/Mamfile
+++ b/src/cmd/ksh93/Mamfile
@@ -1219,7 +1219,7 @@ make install
 							make data/math.tab implicit
 							done data/math.tab
 						done features/math.sh dontcare
-						exec - iffe ${IFFEFLAGS} -v -c "${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS}" ref ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -I${PACKAGE_ast_INCLUDE} -I${INSTALLROOT}/include ${mam_libdll} ${mam_libcmd} ${mam_libast} ${mam_libm} ${mam_libnsl} : run features/math.sh ${PACKAGEROOT}/src/cmd/ksh93/data/math.tab
+						exec - iffe ${IFFEFLAGS} -v -c "${CC} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS}" ref ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -I${PACKAGE_ast_INCLUDE} -I${INSTALLROOT}/include ${mam_libdll} ${mam_libcmd} ${mam_libast} ${mam_libm} ${mam_libnsl} -lm : run features/math.sh ${PACKAGEROOT}/src/cmd/ksh93/data/math.tab
 						make ${PACKAGE_ast_INCLUDE}/ast_standards.h implicit
 						done ${PACKAGE_ast_INCLUDE}/ast_standards.h dontcare
 						make ${INSTALLROOT}/src/lib/libast/FEATURE/float implicit
@@ -1386,7 +1386,7 @@ make install
 		prev +li
 		prev ${mam_libsocket}
 		prev ${mam_libsecdb}
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o ksh pmain.o ${mam_libshell} ${mam_libnsl} ${mam_libast}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o ksh pmain.o -lm ${mam_libshell} ${mam_libnsl} ${mam_libast}
 	done ksh generated
 	make shcomp
 		make shcomp.o
@@ -1408,7 +1408,7 @@ make install
 		prev +li
 		prev ${mam_libsocket}
 		prev ${mam_libsecdb}
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o shcomp shcomp.o ${mam_libshell} ${mam_libnsl} ${mam_libast}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o shcomp shcomp.o -lm ${mam_libshell} ${mam_libnsl} ${mam_libast}
 	done shcomp generated
 	make suid_exec
 		make suid_exec.o
@@ -1429,7 +1429,7 @@ make install
 		prev +li
 		prev ${mam_libsocket}
 		prev ${mam_libsecdb}
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} -lm ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o suid_exec suid_exec.o ${mam_libast} ${mam_libnsl} ${mam_libast}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o suid_exec suid_exec.o -lm ${mam_libast} ${mam_libnsl} ${mam_libast}
 	done suid_exec generated
 	make shell
 		prev libshell.a archive

--- a/src/cmd/ksh93/Mamfile
+++ b/src/cmd/ksh93/Mamfile
@@ -1386,7 +1386,7 @@ make install
 		prev +li
 		prev ${mam_libsocket}
 		prev ${mam_libsecdb}
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o ksh pmain.o -lm ${mam_libshell} ${mam_libnsl} ${mam_libast}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o ksh pmain.o ${mam_libshell} ${mam_libnsl} ${mam_libast} -lm
 	done ksh generated
 	make shcomp
 		make shcomp.o
@@ -1408,7 +1408,7 @@ make install
 		prev +li
 		prev ${mam_libsocket}
 		prev ${mam_libsecdb}
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o shcomp shcomp.o -lm ${mam_libshell} ${mam_libnsl} ${mam_libast}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o shcomp shcomp.o ${mam_libshell} ${mam_libnsl} ${mam_libast} -lm
 	done shcomp generated
 	make suid_exec
 		make suid_exec.o
@@ -1429,7 +1429,7 @@ make install
 		prev +li
 		prev ${mam_libsocket}
 		prev ${mam_libsecdb}
-		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o suid_exec suid_exec.o -lm ${mam_libast} ${mam_libnsl} ${mam_libast}
+		exec - ${CC} ${CCLDFLAGS} ${mam_cc_FLAGS} ${KSH_RELFLAGS} ${KSH_SHOPTFLAGS} ${CCFLAGS} ${LDFLAGS} ${mam_cc_L+-L.} ${mam_cc_L+-L${INSTALLROOT}/lib} -o suid_exec suid_exec.o ${mam_libast} ${mam_libnsl} ${mam_libast} -lm
 	done suid_exec generated
 	make shell
 		prev libshell.a archive

